libmamba/include/mamba/solver/libsolv/database.hpp

```diff
@@ -74,6 +74,7 @@ namespace mamba::solver::libsolv
             const std::string& channel_id,
             PipAsPythonDependency add = PipAsPythonDependency::No,
             UseOnlyTarBz2 only_tar = UseOnlyTarBz2::No,
+            bool verify_packages = false,
```

Comment:
Let's make this a strongly typed boolean to be consistent with the other two parameters.